be/src/exprs/time_functions.cpp

```diff
@@ -982,24 +983,34 @@ StatusOr<ColumnPtr> TimeFunctions::to_unix_from_datetime(FunctionContext* contex
             result.append_null();
         } else {
             timestamp = timestamp < 0 ? 0 : timestamp;
-            timestamp = timestamp > INT_MAX ? 0 : timestamp;
+            timestamp = timestamp > MAX_UNIX_TIMESTAMP ? 0 : timestamp;
             result.append(timestamp);
         }
     }
 
     return result.build(ColumnHelper::is_all_const(columns));
 }
 
+StatusOr<ColumnPtr> TimeFunctions::to_unix_from_datetime(FunctionContext* context, const Columns& columns) {
```

Comment:
Can we define a macro to implement those similar wrapper functions?

Reply:
we have that macro in header file.  yes, I agreee to have macro for implemention functions.

Reply:
Oh, I get it. But since there are not much cases, I think we'd better to leave it to future.

If there are more and more these cases, I think we can do some refactor.